### PR TITLE
[chore] Remove logrus from circleci, filesystem, gitlab, and s3 sources

### PR DESCRIPTION
**Pull request by mcastorina**

### CODE REVIEW

File: pkg/engine/s3.go [hunk 1]
```diff
@@ -46,6 +46,10 @@ func (e *Engine) ScanS3(ctx context.Context, c sources.Config) error {
 	}
 
 	s3Source := s3.Source{}
+	ctx = context.WithValues(ctx,
+		"source_type", s3Source.Type(),
+		"name", "trufflehog - gitlab",
```
Comment (dustin-decker) [inline pkg/engine/s3.go:51]: This should be S3

Comment (dustin-decker) [reply pkg/engine/s3.go:51]: would it be better to say `<name> source`?

File: pkg/sources/gitlab/gitlab.go [hunk 2]
```diff
@@ -383,35 +384,35 @@ func (s *Source) scanRepos(ctx context.Context, chunksChan chan *sources.Chunk)
...
 	return nil
 }
 
-func (s *Source) ignoreRepo(r string) bool {
+func (s *Source) ignoreRepo(ctx context.Context, r string) bool {
 	for _, ignore := range s.ignoreRepos {
 		g, err := glob.Compile(ignore)
 		if err != nil {
-			log.WithError(err).Errorf("could not compile ignore repo glob %s", ignore)
+			ctx.Logger().V(2).Info("could not compile ignore repo glob", "glob", ignore, "error", err)
```
Comment (dustin-decker) [inline pkg/sources/gitlab/gitlab.go:411]: This should be a higher level IMO

Comment (dustin-decker) [reply pkg/sources/gitlab/gitlab.go:411]: It's error or warning kind of information I think

**Review by dustin-decker — APPROVED** [review]
Thanks, Miccah!

File: pkg/engine/s3.go (hunk 1 above)
Comment (mcastorina) [reply pkg/engine/s3.go:51]: Ah, nice catch!

> would it be better to say `<name> source`?

I was just copying the name that we pass to the source, but I think either one is good. Would that be `s3 source` in this case?

Comment (mcastorina) [reply pkg/engine/s3.go]: I decided to make the key `"source_name"` and the value just `"s3"`.

**Pull request merged**